spec/spec.md

```diff
@@ -214,6 +220,8 @@ A [DID controller](https://www.w3.org/TR/did-core/#did-controller) ****MAY**** b
 If present, a DID controller ****MUST**** be represented as a `_cnt._did` record in the form of a comma-separated
 list of controller DID identifiers.
 
+The controller for the [[ref:Identity Key]] Verification Method ****MUST**** match the controller property.
```

Comment:
@frankhinek added this

Reply:
@decentralgabe As a consequence of this additional language, should [Test Vector 2](https://github.com/TBD54566975/did-dht-method/blob/e1b78904bc71cc90600a916244aa7e0d0109f1cc/spec/spec.md?plain=1#L981) have the `controller` property for the Identity Key's verification method set to `did:example:abcd`

Reply:
yes, will update

Reply:
https://github.com/TBD54566975/did-dht-method/pull/102/files#r1462203718

Reply:
@decentralgabe What are your thoughts on expanding the note "The controller for the Identity Key Verification Method **MUST** match the controller property" to provide more context for why this is a requirement?

I don't think there's anything technically stopping the controller from being any other DID, including one that isn't `did:dht` -- but I'd imagine the rationale for this requirement is related to the [DID Core Spec](https://www.w3.org/TR/did-core/#did-controller) description of the intent of the `controller` propertys:

> A [DID controller](https://www.w3.org/TR/did-core/#dfn-did-controllers) is an entity that is authorized to make changes to a [DID document](https://www.w3.org/TR/did-core/#dfn-did-documents).

And the additional note:

> This is particularly important for key recovery in the case of cryptographic key loss, where the [DID subject](https://www.w3.org/TR/did-core/#dfn-did-subjects) no longer has access to their keys, or key compromise, where the [DID controller](https://www.w3.org/TR/did-core/#dfn-did-controllers)'s trusted third parties need to override malicious activity by an attacker.

Since loss of control or compromise of the private key associated with the Identity Key is irrecoverable when it comes to storing mutable items in mainline DHT.

Curious if that's what you had in mind and if there are any other considerations to keep in mind?

Reply:
Yes, exactly, it's a way of ensuring the controller of the DID document can actually control the DID document.

Though I can see a case to be made for changing the controller of the identity key especially when rotating to a new DID. I'm inclined to keep it as-is for now and add more context, what do you think?

Reply:
I do think it makes sense to add some context since most won't have read the DID Core Spec 100 times 🙃 

On that note, for DHT DID documents, does it even make sense to allow for multiple controllers?  Its obviously permitted by the DID Core spec?

I suppose one reason I could come up for permitting it is if the Identity Key of a `did:dht` DID was a threshold key... there could theoretically be many other keys that are the "controller" of that DID... in that they can update the DID document if the threshold is met.  Since anyone can technically do this regardless of what we do in DID DHT Spec, perhaps we allow multiple controllers?

Reply:
yes, adjusted the language to make this possible